base_maintenance/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Base Maintenance",
+    "version": "13.0.1.0.0",
+    "author": "ForgeFlow," "Odoo Community Association (OCA)",
```

Comment:
Better if this is only one string. `"ForgeFlow, Odoo Community Association (OCA)",`